services/api/package.json

```diff
@@ -9,7 +9,7 @@
     "lint"
   ],
   "scripts": {
-    "start": "nodemon -w src --exec \"MOCK_EMAIL=true NODE_ENV=development node src/index.js | pino-pretty --messageKey=message\"",
```

Comment:
so my understanding is that `NODE_ENV=production` carries special meaning but `NODE_ENV=development` does not which is why it is removed here?

also `ENV_NAME` will only be accessible by a config and not available through `process.env`, or does `bedrockio/config` set that as well?

Reply:
```so my understanding is that NODE_ENV=production carries special meaning but NODE_ENV=development does not which is why it is removed here?```

correct.

```also ENV_NAME will only be accessible by a config and not available through process.env, or does bedrockio/config set that as well?
```

It will only be available via process.env if its set as an env var.
So to get it consistently one has to access via `bedrockio/config` (e.g. in development i dont set the ENV_NAME).

Do you think that a problem? I am thinking its better getting all env vars via bedrockio/config otherwise what is the point.

Reply:
not necessarily a problem just trying to figure out how it works